experiments/compositions/composition/api/v1alpha1/plan_types.go

```diff
@@ -33,19 +34,37 @@ type PlanSpec struct {
 	Stages map[string]Stage `json:"stages,omitempty"`
 }
 
+type ResourceStatus struct {
+	Group     string `json:"group,omitempty"`
+	Version   string `json:"version,omitempty"`
+	Kind      string `json:"kind"`
+	Namespace string `json:"namespace,omitempty"`
+	Name      string `json:"name,omitempty"`
+	Status    string `json:"status,omitempty"`
+	IsHealthy bool   `json:"isHealthy"`
```

Comment:
Can we please move to an enum? A) true/false *always* seems to end up confusing at some point. Healthy/Unhealthy doesn't. B) Sooner or later we discover a third (/fourth) state we want. Could be Unknown or Waiting or ....